include/internal/refcount.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+#ifndef HEADER_INTERNAL_REFCOUNT_H
+# define HEADER_INTERNAL_REFCOUNT_H
+
+# if __STDC_VERSION__ >= 201112L && !defined(__STDC_NO_ATOMICS__)
+# include <stdatomic.h>
+# define HAVE_C11_ATOMICS
+# endif
+
+# if HAVE_C11_ATOMICS && ATOMIC_INT_LOCK_FREE > 0
+
+#define HAVE_ATOMICS 1
```

Comment:
Formatting nit. There should be space between `#` and define. And few similar occurrences below. I wish there was a way to omit modifications resulting from formatting nits from CI. It's just waste, not only resources but even waiting... Maybe it's possible to have travis examine last commit for some keyword and skip over whole thing?